### DISCUSSION TITLE
How to build diesel locally only for SQLite

### DISCUSSION THREAD

**Discussion by keepsimple1** [diesel-rs/diesel#2806] [Q&A] [answered]
Hi,
I'm trying to hack Diesel a bit (i.e. trying to add some patch), and wanted to only play with SQLite3 at the beginning. Is there a way to build diesel only for SQLite3?

**Comment by 0xPoe** [comment, marked as answer]
Try `cargo build --no-default-features --features "sqlite"` in deisel_cli.

**Reply by 0xPoe** [reply]
You can find more commands about compiling here https://github.com/diesel-rs/diesel/blob/master/bin/test. Also thanks for your interest in trying to add patch!

**Reply by keepsimple1** [reply]
> Try `cargo build --no-default-features --features "sqlite"` in deisel_cli.

Thanks! Btw, looks like this works inside `diesel/` sub crate, not at the workspace root.
